src/main/java/lotto/code/RankCode.java

```diff
@@ -0,0 +1,48 @@
+package lotto.code;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public enum RankCode {
+	FIRST(6, 2000_000_000),
```

Comment:
언더 스코어 하나를 놓치신 것 같아요 ! :)

```suggestion
	FIRST(6, 2_000_000_000),
```